modules/chat/client/config/chat.client.routes.js

```diff
@@ -15,8 +15,7 @@
         controller: 'ChatController',
         controllerAs: 'vm',
         data: {
-          roles: ['user', 'admin'],
-          pageTitle: 'Chat'
```

Comment:
After looking back at the directive changes, I understand what you're doing here a bit better. I misunderstood the inclusion of the `pageTitle` option in the other routes for Articles, as an "override". But here you're still using the State name to build the resulting title.